scrutiny.json

```diff
@@ -0,0 +1,82 @@
+                    ],
+                    [
+                        "--privileged",
+                        ""
+                    ]
+                ],
+                "ports" : {
+                    "8080": {
+                        "description": "Port for scrutiny's web interface and API.",
+                        "host_default": 8080,
+                        "label": "WebUI HTTP port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Scrutiny config files. Eg: create a Share called Scrutiny-Share-Config for this purpose alone.",
+                        "label": "Config share"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid system UserID for Scrutiny. Must have full permissions on the previous Share.",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid system GroupID for Scrutiny. This group or the above UserID must have full permissions on the previous Share.",
+                        "label": "GID",
+                        "index": 2
+                    },
+                    "SCRUTINY_API_ENDPOINT": {
+                        "description": "API endpoint of the scrutiny UI. Do not change unless using as a remote collector. Default: 'http://localhost:8080'",
+                        "label": "SCRUTINY_API_ENDPOINT",
+                        "index": 3
+                    },
+                    "SCRUTINY_WEB": {
+                        "description": "Run the web service. Default: 'true'",
+                        "label": "SCRUTINY_WEB",
+                        "index": 4
+                    },
+                    "SCRUTINY_COLLECTOR": {
+                        "description": "Run the metrics collector. Default: 'true'",
+                        "label": "SCRUTINY_COLLECTOR",
+                        "index": 5
+                    }
+                }
+            }
+        },
+        "description": "<p><p><a href='https://github.com/AnalogJ/scrutiny'>Scrutiny</a> WebUI for smartd S.M.A.R.T monitoring. Scrutiny is a Hard Drive Health Dashboard &amp; Monitoring solution, merging manufacturer provided S.M.A.R.T metrics with real-world failure rates from Backblaze.</p></p><p><strong>Note:</strong>This image is updating the stats only once a day, if you see: '<i>No Devices Detected!</i>' after the installation, just wait 24 hours. <br>This process can also be triggered manually from the shell by executing the following command: <code>docker exec scrutiny scrutiny-collector-metrics run</code><br>More documentation on this can be found here: <a href='https://github.com/AnalogJ/scrutiny#usage'>Scrutiny GitHub</a></p>",
```

Comment:
Could you move the Note (</p></p><p><strong>Note:</strong>This image is updating the stats only...) to the additional info tag. I think the "more documentation" line could stay in the description?

Finally, add to the end of the description something like this:
<p>Based on custom docker image: <a href='https://hub.docker.com/r/linuxserver/scrutiny' target='_blank'>https://hub.docker.com/r/linuxserver/scrutiny</a>, available for amd64, arm32v7, and arm64v8</p>",

this will clarify what image is used (and the link to it) as well as highlight the multi-architecture availability (until @FroggyFlox devises a comprehensive way to highlight available architectures).